packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -3132,14 +3131,20 @@ Future<void> _analyzeProject(String workingDir, { List<String> expectedFailures
     }
   }
   final String stdout = exec.stdout.toString();
-  final List<String> errors;
-  final RegExp regex = RegExp(_kLintingErrorPattern, multiLine: true);
+  final List<String> errors = <String>[];
   try {
-    errors = const LineSplitter()
-        .convert(stdout)
-        .where((String line) => regex.hasMatch(line))
-        .map(lineParser)
-        .toList();
+    bool analyzeLineFound = false;
+    const LineSplitter().convert(stdout).forEach((String line) {
+      if (!analyzeLineFound && line.startsWith('Analyzing flutter_project')) {
```

Comment:
nit: can you add a comment explaining we want to ignore all output from pub get?